assets/global.js

```diff
@@ -679,17 +679,16 @@ class SlideshowComponent extends SliderComponent {
   }
 
   setAutoPlay() {
-    this.sliderAutoplayButton = this.querySelector('.slideshow__autoplay');
+    if (this.querySelector('.slideshow__autoplay') !== null) {
```

Comment:
Do we need to check for this ? We only run this `setAutoPlay()` function if this is true: 
```javascript
if (this.slider.getAttribute('data-autoplay') === 'true') this.setAutoPlay();
```
Are we thinking of doing autoplay without a pause button ?